docker/test_env/geth.go

```diff
@@ -71,20 +76,20 @@ func (g *Geth) StartContainer() (blockchain.EVMNetwork, InternalDockerUrls, erro
 	if err != nil {
 		return blockchain.EVMNetwork{}, InternalDockerUrls{}, err
 	}
-	httpPort, err := ct.MappedPort(context.Background(), "8544/tcp")
+	httpPort, err := ct.MappedPort(context.Background(), exposedHttpPort)
 	if err != nil {
 		return blockchain.EVMNetwork{}, InternalDockerUrls{}, err
 	}
-	wsPort, err := ct.MappedPort(context.Background(), "8545/tcp")
+	wsPort, err := ct.MappedPort(context.Background(), exposedWsPort)
 	if err != nil {
 		return blockchain.EVMNetwork{}, InternalDockerUrls{}, err
 	}
 
 	g.Container = ct
 	g.ExternalHttpUrl = fmt.Sprintf("http://%s:%s", host, httpPort.Port())
-	g.InternalHttpUrl = fmt.Sprintf("http://%s:8544", g.ContainerName)
+	g.InternalHttpUrl = fmt.Sprintf("http://%s:%s", g.ContainerName, httpPort)
```

Comment:
```suggestion
	g.InternalHttpUrl = fmt.Sprintf("http://%s:%s", g.ContainerName, httpPort.Port())
```

Reply:
This is actually wrong. I had a constant "httpPort" that was getting overridden 🤦 Fixed now